app/logger/logger.go

```diff
@@ -0,0 +1,43 @@
+package logger
+
+import (
+	"log"
+
+	"go.uber.org/zap"
+)
+
+type Wrapper struct{ log *zap.SugaredLogger }
```

Comment:
Let's call it Logger 😄 cos it's a logger.

Reply:
Deal 😄